tests/test_functions.py

```diff
@@ -114,6 +114,21 @@ def test_extract_hashbits_info():
             print >>sys.stderr, '...failed to remove {fn}'.format(fn)
 
 
+def check_file_status_kfile():
+    try:
+        kfile.check_file_status('thisfiledoesnotexistatall')
```

Comment:
put an 'assert False' after this, because it *must* raise an OSError to succeed.